ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6618,3 +6618,16 @@ VMADC:
   - mixed_air_damper_percentage_sensor
   implements:
   - CONTROL
+
+PSM2X:
```

Comment:
This seems to just be a combination of SS and WDPM for two booster pumps. See more in the comment below.